src/yz_index_hashtree.erl

```diff
@@ -639,7 +640,15 @@ get_all_locks(Type, Pid) ->
     %% file page cache but the bg manager stuff is kind of convoluted
     %% and there isn't time to figure this all out for 2.0. Thus,
     %% Yokozuna will not bother with the Solr lock for now.
-    ok == yz_entropy_mgr:get_lock(Type, Pid).
+    try yz_entropy_mgr:get_lock(Type, Pid) of
+        ok -> true;
+        Other ->
+            lager:debug("Could not get lock: ~p", [Other]),
+            false
+    catch exit:{timeout,_} ->
```

Comment:
I wish we have a more "common" abstraction for this specific exit, since it's possible to catch this for any handle_call w/ a timeout.